lib/msf/core/web_services/servlet/service_servlet.rb

```diff
@@ -10,7 +10,6 @@ def self.api_path_with_id
 
   def self.registered(app)
     app.get  ServiceServlet.api_path, &get_services
```

Comment:
Also would the following definition of `self.get_services` still be needed? Thought we were depreciating that code. If I'm confused then my apologies.

Reply:
So this is down to a confusing naming convention. Methods in this file are named with the pattern `[HTTP VERB]_[ENDPOINT]`. So `get_services` actually means, "the code that should be run when the `GET` verb is used on the `services` endpoint".

I'm removing this endpoint because the `services` method shouldn't take IDs in the URL, not because it's what get's hit when `framework.db.get_services` is called by a module.

Reply:
This did make be go back, double check, and remove some more unneeded code though, so it's still a good shout 👍

Reply:
@adamgalway-r7 Ah I thought we were removing the `get_services` method in its entirety so thats why I was confused and its also why I mentioned `self.get_services` and questioned why this wasn't being removed. This makes more sense now.